src/mainwindow.cpp

```diff
@@ -1181,7 +1170,6 @@ void MainWindow::setupDatabases()
         QFile noteDBFile(noteDBFilePath);
         noteDBFile.rename(dir.path() + QDir::separator() + QStringLiteral("oldNotes.db"));
         {
-            QFile noteDBFile(noteDBFilePath);
```

Comment:
You can't just remove this line as far as I understand. That file was moved, and we want to open the new location for writing and not overwrite `oldNotes.db`. Likely the explicit new scope existed exactly for the purpose of introducing this new variable.

We could likely still reuse the variable, if we call `setFileName(noteDBFilePath)` on it after the call to `rename`. The extra scope can be removed as well.

Reply:
You're absolutely right. It was a complete documentation misread on my part. Sorry!